website/docs/d/oracle_autonomous_database_cross_region_disaster_recovery.html.markdown

```diff
@@ -0,0 +1,129 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_autonomous_database"
+description: |-
+  Gets information about an existing Autonomous Database.
```

Comment:
The description should be "Gets information about an existing Cross Region Disaster Recovery Autonomous Database."